sites/main-site/src/content/special-interest-groups/immunology/index.md

```diff
@@ -18,12 +18,20 @@ pipelines:
 
 # Introduction
 
-TBD
+The nf-core Special Interest Group Immunology aims to connect researchers and developers working on immunology-related pipelines within the nf-core community. While several pipelines relevant to immunology already exist, the community around them remains fragmented. This SIG seeks to bring together users and contributors to foster collaboration and knowledge exchange.
+Our goals include facilitating discussions on current nf-core immunology pipelines, identifying gaps in existing workflows, and defining requirements for new pipelines and tools. By creating a dedicated space for questions and idea-sharing, we hope to strengthen the nf-core immunology community and support the development of high-quality, reusable bioinformatics solutions.
+Contact: [Mark Polster](https://nfcore.slack.com/team/U05SAA3HHL3) & [Jonas Scheid](https://nfcore.slack.com/team/U03DN4C1JQ5)
 
 # Activities
 
-TBD
+The SIG Immunology will host regular meetings to facilitate discussions on immunology-related nf-core pipelines and tools. Each session will feature a pre-selected topic and a presentation to encourage in-depth discussions. Community input is highly valued, and everyone is welcome to suggest topics and provide feedback.
```

Comment:
Please replace all occurrences of the sig initialism with the written out version

Reply:
Because we try to minimize the usage of jargon on nf-core, especially for community initiatives. By avoiding using it here hopefully helps us also to not use it in other places.